### PR TITLE
Add GROUND_GPS message

### PR DESCRIPTION
**Pull request by fvantienen**
This adds a message with a ground GPS which is forwarded to the aircraft from the GCS.

### CODE REVIEW

File: message_definitions/v1.0/messages.xml [hunk 1]
```diff
@@ -2480,6 +2480,18 @@
       <field name="course"  type="float" unit="deg"/>
     </message>
 
+    <message name="GROUND_GPS" id="57" link="forwarded">
```
Comment (gautierhattenberger) [inline message_definitions/v1.0/messages.xml]: I might be interested with the same kind of information for other kinds of target. What do you think about a more generic message (like `GROUND_TARGET`, `TARGET_GPS`, whatever) and add an identifier field, eventually with a reserved value like 0 for the GCS ?

Comment (fvantienen) [reply message_definitions/v1.0/messages.xml]: That is fine for me also ;)

Comment (dewagter) [reply message_definitions/v1.0/messages.xml]:
```suggestion
    <message name="TARGET_POS" id="57" link="forwarded">
```

So far GPS in message names is only used for sensors. Do you mind using POS or LOCATION.

File: message_definitions/v1.0/messages.xml [hunk 2]
```diff
@@ -2480,6 +2480,18 @@
...
+      <description>
+        Ground station GPS position for functions like Follow Me
+      </description>
+      <field name="ac_id" type="uint8"/>
```
Comment (dewagter) [inline message_definitions/v1.0/messages.xml]:
```suggestion
      <field name="ac_id" type="uint8"/>
      <field name="target_id" type="uint8"/>
```

File: message_definitions/v1.0/messages.xml [hunk 3]
```diff
@@ -2480,6 +2480,19 @@
       <field name="course"  type="float" unit="deg"/>
     </message>
 
+    <message name="TARGET_POS" id="57" link="forwarded">
+      <description>
+        Ground station GPS position for functions like Follow Me
```
Comment (gautierhattenberger) [inline message_definitions/v1.0/messages.xml]: The description should be changed as well as it is not limiter to ground station

**Pull request merged**